packages/cactus-plugin-keychain-azure-kv/src/main/json/openapi.json

```diff
@@ -0,0 +1,81 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Hyperledger Cactus - Keychain API",
+    "description": "Contains/describes the Keychain API types/paths for Hyperledger Cactus.",
+    "version": "0.3.0",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "components": {
+    "schemas": {
+    }
+  },
+  "paths": {
+    "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-azure-kv/get-keychain-entry": {
+      "post": {
+        "x-hyperledger-cactus": {
+          "http": {
+            "path": "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-azure-kv/get-keychain-entry",
+            "verbLowerCase": "post"
+          }
+        },
+        "operationId": "getKeychainEntryV1",
+        "summary": "Retrieves the contents of a keychain entry from the backend.",
+        "parameters": [],
+        "requestBody": {
+          "$ref": "../../../../cactus-core-api/src/main/json/openapi.json#/components/requestBodies/keychain_get_entry_request_body"
+        },
+        "responses": {
+          "200": {
+            "$ref": "../../../../cactus-core-api/src/main/json/openapi.json#/components/responses/keychain_get_entry_200"
```

Comment:
@elenaizaguirre @jagpreetsinghsasan Please coordinate your PRs merging in order to make sure that the openapi.json spec references are updated "at the right time" depending on who's PR gets merged first.

Context: Elena is working on a change that necessitates for us to use the http URLs instead of relative file paths.